website/components/Layout/index.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import Head from 'components/Head';
+import Navbar from 'components/Navbar';
+import StyledLink from 'components/StyledLink';
+import PropTypes from 'prop-types';
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 4fr 1fr;
+  gap: 2rem;
+  padding: 1.5rem;
+`;
+
+const Footer = styled.footer`
+  display: flex;
+  gap: 2rem;
+  padding: 38px 0;
+`;
+
+const Layout = ({ children, meta, navigationContent }) => {
```

Comment:
This component should use the [Layout](https://design-system-git-ds-website-strapijs.vercel.app/?path=/story/design-system-components-layout--base) one. Can you check the example and update your code accordingly?